docs/api/event-definitions-map.md

```diff
@@ -10,13 +10,10 @@ anotherEventDefinition,
 
 const eventDefinitionsMap = {
   'SOME_ACTION_TYPE': someEventDefinition,
-  'SOME_OTHER_ACTION_TYPE': [someOtherEventDefinition, anotherEventDefinition],
```

Comment:
We can definitely create a helper function to do this. Given an array of event definitions it would return a single event definition. Great idea!